nautobot/docs/release-notes/version-2.1.md

```diff
@@ -12,3 +12,7 @@ The `Job` base class now includes a [`create_file(filename, content)`](../develo
 
 !!! info
     The specific storage backend used to retain such files is controlled by the [`JOB_FILE_IO_STORAGE`](../user-guide/administration/configuration/optional-settings.md#job_file_io_storage) settings variable. The default value of this setting uses the Nautobot database to store output files, which should work in all deployments but is generally not optimal and better alternatives may exist in your specific deployment. Refer to the documentation link above for more details.
+
+#### External Integration Model ([#4694](https://github.com/nautobot/nautobot/issues/4694))
+
+A new [`ExternalIntegration` model](../user-guide/platform-functionality/externalintegration.md) has been added which provides a centralized store for data such as URLs and credentials that are used to access systems external to Nautobot. This information can then be used by jobs or apps to perform actions such as creating DNS records or updating configuration management tickets.
```

Comment:
Would you mind adding `<!-- towncrier release notes start -->` at the end of this file while you're in here? I missed that when I created this file. Guess we should also update the towncrier `filename` in `pyrproject.toml` for `next` as well.